package-lock.json

```diff
@@ -16,25 +16,39 @@
         "@oclif/errors": "^1.1.2",
         "debug": "^4.1.0",
         "jsonwebtoken": "^8.3.0"
+      },
+      "dependencies": {
+        "@adobe/aio-lib-core-config": {
+          "version": "1.2.8",
```

Comment:
@justinedelson  `@adobe/aio-cli-plugin-jwt-auth` is still dependent on config 1.x.x, and https://github.com/adobe/aio-cli-plugin-jwt-auth is archived in favor of `@adobe/aio-cli-plugin-auth` or `@adobe/aio-lib-ims` (recommended to depend on a lib and not a plugin in a plugin)

Reply:
@moritzraho is there a migration guide from aio-cli-plugin-jwt-auth to aio-lib-ims